src/clr-angular/button/button-group/button.ts

```diff
@@ -19,7 +19,8 @@ import { ButtonInGroupService } from '../providers/button-in-group.service';
                 (click)="emitClick()"
                 [attr.type]="type"
                 [attr.name]="name"
-                [attr.disabled]="disabled">
+                [attr.disabled]="disabled"
+                [attr.id]="id">
```

Comment:
Nitpick: you can just bind to `[id]` on a native element. That sets the id property instead of the attribute, which in turn sets the attribute. Honestly, I'm not even sure which is supposedly better.

Reply:
Missed this, I prefer properties by default. I think regular html elements will reflect them back and forth but custom elements you have to manually handle this. Angular I think reflects the property to the attribute but not the other way around. I'll get this updated, I think this might change the setter signature a bit.